cmd/agent/subcommands/flare/command.go

```diff
@@ -278,6 +280,26 @@ func makeFlare(flareComp flare.Component,
 		err     error
 	)
 
+	if cliParams.streamLogsDuration <= 0 {
+		return fmt.Errorf("stream-logs-duration must be a positive value")
+	}
+
+	streamLogParams := streamlogs.CliParams{
+		FilePath: commonpath.DefaultStreamlogsLogFile,
+		Duration: cliParams.streamLogsDuration,
+		Quiet:    true,
+	}
+
+	streamLogDuration := 60 * time.Second // default duration
```

Comment:
Since `stream-logs-duration` is defaulted to 60 seconds in the CLI args (above), the check `streamLogParams.Duration != 0` will never be `true` unless the user sets it to `0`, which seems unlikely. 

I would either: 
1. Keep the 60 second default in the `DurationVarP` and abandon the check if this flag is set without `with-stream-logs` (because it has no negative effects, and is an unlikely error case)

2. or - set the default in `DurationVarP` to something like `-1`. Then you can perform a check like `if cliParams.streamLogsDuration != -1` (else apply the default 60 seconds) this way we know the user set it explicitly, in that case you can perform the check if `with-stream-logs` is present. 

Let me know if you want to chat offline to further clarify this :)

Reply:
Since the error generated from the checks also create some issue on our flare e2e tests, I just adjusted the logic so that we force the 60s duration if there is an invalid number, and automatically turn on `--with-stream-logs` . I think this might save a lot of headache for people also